models/pull.js

```diff
@@ -28,6 +28,19 @@ function Pull(data, signatures, comments, commitStatus, labels) {
       this.data.closes = data.closes;
       this.data.connects = data.connects;
    }
+
+   var isPassing = function() {
```

Comment:
Could've been a short function to get implicit this binding, but eh.

Also now that we're running a sorta recent version of node we probably shouldn't be too shy about using `const` instead of `var`.